src/styles/blocks/product.css

```diff
@@ -0,0 +1,82 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product__disc {
+  display: inline-block;
+  font-weight: 500;
+  font-size: 12px;
+  padding: 40px 0 4px;
+  line-height: 18px;
+  margin: 0;
+  color: #060b35;
+}
+
+.product__code {
+  align-self: flex-start;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.product__price {
+  display: flex;
+  align-self: flex-start;
+  margin-top: 24px;
+  justify-content: space-between;
+  line-height: 18px;
+  align-items: center;
+  width: 100%;
+}
+
+.product__text {
+  font-size: 12px;
+  font-weight: 400;
+  margin: 0;
+  color: #616070;
+}
+
+.product__number {
+  font-size: 16px;
+  font-weight: 700;
+  margin: 0;
+  color: #060b35;
+}
+
+.product__buy {
+  margin-top: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  color: #fff;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 16px;
+  padding: 12px 70px;
+  font-weight: 700;
+}
+
+.product__link:hover {
+  border: 1px solid #00acdc;
```

Comment:
It looks like we shouldn't set the full `border property` again for the button hover state. You already defined it for the normal state. Now just the color needs to change on hover. Checklist p8.

Reply:
Done